docker-cloud/cloud-swarm/index.md

```diff
@@ -2,164 +2,27 @@
 description: Work with swarms in Docker Cloud
 keywords: swarm mode, swarms, orchestration Cloud, fleet management
 title: Swarms in Docker Cloud
+notoc: true
 ---
 
-<br>
+Docker Cloud now allows you to connect to clusters of Docker Engines running in v1.13 [swarm mode](/engine/swarm/).
```

Comment:
Let's add a bit more context to the introduction of Swarm mode with something like:

_"Docker Cloud now allows you to easily manage a fleet of swarms by connecting to clusters of Docker Engines running in v1.13 [swarm mode](/engine/swarm/). 

With Beta Swarm mode in Docker Cloud, you can provision Swarms to popular Cloud providers, or register existing Swarms to Docker Cloud. Use your Docker ID to authenticate and securely access personal or team Swarms."_

Also, let's remove the below image.